root/edit/details/EditRelationshipAttribute.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import IntentionallyUntranslatedIcon
+  from '../components/IntentionallyUntranslatedIcon';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+import localizeLinkAttributeTypeName
+  from '../../static/scripts/common/i18n/localizeLinkAttributeTypeName';
+import WordDiff from '../../static/scripts/edit/components/edit/WordDiff';
+
+type EditRelationshipAttributeEditT = {
+  ...EditT,
+  +display_data: {
+    +child_order?: CompT<number>,
+    +description: CompT<string | null>,
+    +name: CompT<string>,
+    +original_description: string | null,
+    +original_name: string,
+    +parent?: CompT<LinkAttrTypeT | null>,
+  },
+};
+
+type Props = {
+  +edit: EditRelationshipAttributeEditT,
+};
+
+const EditRelationshipAttribute = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const childOrder = display.child_order;
+  const oldDescription = display.description.old ?? '';
+  const newDescription = display.description.new ?? '';
+  const descriptionChanges = newDescription !== oldDescription;
+  const name = display.name;
+  const parent = display.parent;
+  const newParentName = parent?.new
+    ? localizeLinkAttributeTypeName(parent.new)
+    : '';
+  console.log(newParentName);
+  const oldParentName = parent?.old
+    ? localizeLinkAttributeTypeName(parent.old)
+    : '';
+  console.log(oldParentName);
+  const rawIconSection = (
+    <>
+      {' '}
+      <IntentionallyUntranslatedIcon />
+    </>
+  );
+
+  return (
+    <table className="details edit-relationship-attribute">
+      {name.new === name.old ? (
+        <tr>
+          <th>{addColonText(l('Name'))}</th>
+          <td colSpan="2">
+            {l_relationships(name.old)}
+          </td>
+        </tr>
+      ) : (
+        <WordDiff
+          extraNew={rawIconSection}
+          extraOld={rawIconSection}
+          label={addColonText(l('Name'))}
+          newText={name.new}
+          oldText={name.old}
+        />
+      )}
+
+      {descriptionChanges ? (
+        <WordDiff
+          extraNew={nonEmpty(newDescription) ? rawIconSection : null}
+          extraOld={nonEmpty(oldDescription) ? rawIconSection : null}
+          label={addColonText(l('Description'))}
+          newText={newDescription}
+          oldText={oldDescription}
+        />
+      ) : nonEmpty(oldDescription) ? (
+        <tr>
+          <th>{addColonText(l('Description'))}</th>
+          <td colSpan="2">
+            {expand2react(l_relationships(oldDescription))}
```

Comment:
Even though _raw_ doesn’t mean anything to me :wink: should this raw description be localized still?

```suggestion
            {expand2react(oldDescription)}
```

Reply:
This one is not raw. This is the one that is shown just for extra info when it hasn't been changed, so localizing it makes sense to me.